CHANGELOG.md

```diff
@@ -3,6 +3,26 @@
 Details changes in each release of EarlGrey. EarlGrey follows
 [semantic versioning](http://semver.org/).
 
+## [1.5.0](https://github.com/google/EarlGrey/tree/1.5.0) (10/31/2016)
+
+```
+Baseline: [d46a15f]
+   + [d46a15f]: Update Info.plist / Podspec / Cheatsheet for EarlGrey 1.4.0
+```
+
+### Compatibility
+* Add iOS 10 Support to Travis tests.
```

Comment:
Travis & Swift 3.0 are enhancements.
Also link Swift 3.0 to the design doc.

Swift 3.0 [design doc](...)